src/test/java/rx/subjects/PublishSubjectTest.java

```diff
@@ -150,6 +160,15 @@ private void assertErrorObserver(Observer<String> observer) {
         verify(observer, never()).onCompleted();
     }
 
+    private void assertNeverErrorObserver(Observer<String> observer) {
+        verify(observer, never()).onNext("one");
+        verify(observer, never()).onNext("two");
+        verify(observer, never()).onNext("three");
+        verify(observer, never()).onNext("four");
+        verify(observer, times(1)).onError(any(Throwable.class));
+        verify(observer, never()).onCompleted();
+    }
```

Comment:
I think you'll want `verifyNoMoreInteractions(observer)` to end these two methods, otherwise sending an `onNext` of "five" would not fail the test.